media/Circletracer/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<!--
+MIT License
+
+Copyright (c) Lutz Roeder
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+This file referenced the result of
+https://github.com/lutzroeder/netron/tree/50083007d427fa0c03eda4ea0e62de373ec89c9b
+-->
+<head>
+    <meta charset="UTF-8">
+    <!--
+        Use a content security policy to only allow loading images from https or from our extension directory,
+        and only allow scripts that have a specific nonce.
+    -->
+    <meta 
+        http-equiv="Content-Security-Policy"
+        content="https: data:;
+                    style-src 'unsafe-inline' _webview.cspSource;
+                    script-src 'nonce-_nonce';">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link href="_styleUri" rel="stylesheet">
+    <script src="_dagreUri" nonce="_nonce"></script>
+    <script src="_loadFileUri" nonce="_nonce"></script>
+    <script src="_treeMapUri" nonce="_nonce"></script>
+    <script src="_makeDataUri" nonce="_nonce"></script>
+    <script src="https://d3js.org/d3.v5.min.js" nonce="_nonce"></script>
+    <title>Circle Tracer</title>
+</head>
+
+<body>
+    <div id="root">
+        <main>
+            <h1 id="title">Circle Tracer</h1>
+            <div id="main">
+                <nav>
+                    <div class="left-btns">
+                        <button id="first-json-btn" class="json-load-btn" onclick="openFileSelector(1);">Load First
```

Comment:
We need something like...

```suggestion
                        <button id="first-json-btn" class="json-load-btn" onclick="openFileSelector("#first-json-btn");">Load First
```

according to https://github.com/Samsung/ONE-vscode/pull/236#discussion_r721860995